src/Http/Client/Auth/Oauth.php

```diff
@@ -220,6 +220,9 @@ protected function _rsaSha1(Request $request, array $credentials): string
             $credentials['privateKeyPassphrase'] = $passphrase;
         }
         $privateKey = openssl_pkey_get_private($credentials['privateKey'], $credentials['privateKeyPassphrase']);
+        if ($privateKey === false) {
+            throw new RuntimeException('Could not open privateKey file. Got ' . openssl_error_string());
```

Comment:
openssl_error_string() is a queue so we might need to pull all the errors to get the last if something else failed first.

Can we just write a helper function and call it if any of these openssl apis fail?

Reply:
> Can we just write a helper function and call it if any of these openssl apis fail?

We could. We could have something like `checkOpenSslError()` that raises on an error.